### DISCUSSION TITLE
Donut Chart: How to show label in outside ?

### DISCUSSION THREAD

**Discussion by rubysky17** [apexcharts/apexcharts.js#3799] [Q&A]
I want to show dataLabels outside donut chart and it connect a chart with line. UI like this: 
![image](https://user-images.githubusercontent.com/69248909/235124683-8e704861-b767-4e02-83ba-291883045b4e.png)

- i used annotations and custom datalbels but not work :( thanks all.

**Comment by echipachenko** [comment]
Any updates on this?

**Comment by xHypnos** [comment]
I have the same question

**Comment by flyingduck92** [comment]
i am also curious about this
